tests/python-tests/CWE-915/remote/MassAssignment.expected

```diff
@@ -0,0 +1,79 @@
+| massassignment.py:13:26:13:28 | ControlFlowNode for key | semmle.label | ControlFlowNode for key |
+| massassignment.py:19:9:19:15 | ControlFlowNode for request | semmle.label | ControlFlowNode for request |
+| massassignment.py:23:19:23:25 | ControlFlowNode for request | semmle.label | ControlFlowNode for request |
+| massassignment.py:23:19:23:30 | ControlFlowNode for Attribute | semmle.label | ControlFlowNode for Attribute |
+| massassignment.py:23:19:23:39 | ControlFlowNode for Attribute() | semmle.label | ControlFlowNode for Attribute() |
+| massassignment.py:23:42:23:48 | ControlFlowNode for request | semmle.label | ControlFlowNode for request |
+| massassignment.py:26:22:26:28 | ControlFlowNode for request | semmle.label | ControlFlowNode for request |
+| massassignment.py:26:22:26:33 | ControlFlowNode for Attribute | semmle.label | ControlFlowNode for Attribute |
+| massassignment.py:26:22:26:42 | ControlFlowNode for Attribute() | semmle.label | ControlFlowNode for Attribute() |
+| massassignment.py:26:45:26:51 | ControlFlowNode for request | semmle.label | ControlFlowNode for request |
+| massassignment.py:29:14:29:20 | ControlFlowNode for request | semmle.label | ControlFlowNode for request |
+| massassignment.py:29:14:29:25 | ControlFlowNode for Attribute | semmle.label | ControlFlowNode for Attribute |
+| massassignment.py:29:14:29:34 | ControlFlowNode for Attribute() | semmle.label | ControlFlowNode for Attribute() |
+| massassignment.py:47:9:47:18 | SSA variable param_name | semmle.label | SSA variable param_name |
+| massassignment.py:47:9:47:31 | ControlFlowNode for Tuple | semmle.label | ControlFlowNode for Tuple |
+| massassignment.py:47:9:47:31 | ControlFlowNode for Tuple [Tuple element at index 0] | semmle.label | ControlFlowNode for Tuple [Tuple element at index 0] |
+| massassignment.py:47:9:47:31 | IterableElement | semmle.label | IterableElement |
+| massassignment.py:47:9:47:31 | IterableSequence | semmle.label | IterableSequence |
+| massassignment.py:47:36:47:42 | ControlFlowNode for request | semmle.label | ControlFlowNode for request |
+| massassignment.py:47:36:47:47 | ControlFlowNode for Attribute | semmle.label | ControlFlowNode for Attribute |
+| massassignment.py:49:23:49:32 | ControlFlowNode for param_name | semmle.label | ControlFlowNode for param_name |
+| massassignment.py:57:8:57:14 | ControlFlowNode for request | semmle.label | ControlFlowNode for request |
+| massassignment.py:58:16:58:22 | ControlFlowNode for request | semmle.label | ControlFlowNode for request |
+| massassignment.py:61:21:61:27 | ControlFlowNode for request | semmle.label | ControlFlowNode for request |
+| massassignment.py:61:21:61:32 | ControlFlowNode for Attribute | semmle.label | ControlFlowNode for Attribute |
+| massassignment.py:65:13:65:16 | SSA variable item | semmle.label | SSA variable item |
+| massassignment.py:67:27:67:30 | ControlFlowNode for item | semmle.label | ControlFlowNode for item |
+subpaths
+#select
```

Comment:
There is an issue using remote-flows for `flask.request`. This is a known issue and will be resolved in `codeql/python-queries` lib qlpack